solution_output.Rmd

```diff
@@ -1,16 +0,0 @@
----
-title: "Solution Test"
```

Comment:
Checking my understanding - this can be removed because there are 4 Rmd files included as part of testing - 1 input and then 3 that serve as a check for each situation (e.g., totally blank).

Reply:
Correct. That was my informal test doc and should probably never have been committed. `check()` complained about it. 🙂